pipelines/components/image_classifier_keras/README.md

```diff
@@ -0,0 +1,131 @@
+
+Unless required by applicable law or agreed to in writing, software<br/>
+distributed under the License is distributed on an "AS IS" BASIS,<br/>
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.<br/>
+See the License for the specific language governing permissions and<br/>
+limitations under the License.<br/>
+
+## Overview
+
+This *ML pipeline* performs image classification on images using a Convolutional
+Neural Network (CNN) in Keras.
+
+## Intended Use
+
+### Primary Use
+
+Users of this template can preprocess and train collections of images for
+classification. The trained classifier is saved as a keras model, which can then
+be deployed into an application.
+
+
+### Example Use Cases
+
+This pipeline comes with two example use cases (Datasets): EMNIST and
+Fruits-360. Both datasets have been used in competitions in Kaggle.
+
+The EMNIST dataset is an extension of the MNIST dataset. It include
+all lower and uppercase letters, along with digits, in the same 28x28 grayscale
+format as MNIST. The dataset demonstrates classification of grayscale images in
+a larger classification range (62 classes) and substantial size images in the
+training set (800,000).
+
+The Fruits-360 are color images of fruits and their varieties. The dataset
+demonstrates classification involving subtle features, particularly in color shading
+and texture of fruits of the same type (e.g., apples), but of different variety.
+
+## Input
+
+### Data
+
+The pipeline supports training of JPG, PNG, and TIF images, in either color
+(RGB) or grayscale (K).
+
+Datasets must be in a directory structure, where the subdirectories
+are the corresponding classes, and their contents are the images. The following
+two directory layouts are recognized (supported):
+
+*Combined Dataset (Training and Test)*
+
+                       root
+               /        |          \
+              V         V           V
+        class_1     class_2     ... class_n
+        image(s)    image(s)        image(s)
+
+*Separated Dataset*
+
+                       root
+                    /         \
+                   V           V
+            Training            Test
+              /   \             /     \
+            V      V           V       V
+       class_1 ... class_N   class_1 .. class_n
+       image(s)    image(s)  image(s)   image(s)
+
+### Sample Datasets
+
+EMNIST: gs://cloud-samples-data/air/fruits360/fruits360.zip
+
+FRUITS360: gs//cloud-samples-data/air/fruits360/fruits360.zip
+
+
+## Run-Time Parameters
+
+### Required
+
+      --name                the name of the dataset (emnist and fruits360 are reserved for example datasets)
+      --training_data_path  the path to the training data
+
+### Optional
+
+      --test_data_path      the path to the test data (if not specified, 20% of training data is used as test).
+      --gcs                 the location on GCS storage of the dataset to download and unzip.
```

Comment:
It seems the user is expected to place their dataset in a GCS bucket. Is that the only option? Can the user place the inout files in a local directory?

Reply:
They can do either (GCS or local). There is an example of both in the README.md